kinisi/diffusion.py

```diff
@@ -135,12 +137,33 @@ def diffusion(self, start_dt: sc.Variable, **kwargs):
         self.bayesian_regression(start_dt=start_dt, **kwargs)
         self._diffusion_coefficient = sc.to_unit(self.gradient / (2 * self.msd.coords['dimensionality'].value), 'cm2/s')
 
+
+    def jump_diffusion(self, start_dt: sc.Variable, **kwargs):
+        """
+        Calculation of the diffusion coefficient. 
+
+        :param start_dt: The time at which the diffusion regime begins.
+        :param kwargs: Additional keyword arguments to pass to :py:func:`bayesian_regression`.
+        """
+
+        self.bayesian_regression(start_dt=start_dt, **kwargs)
+        # * self.msd.coords['n_samples'][0].value
+        self._jump_diffusion_coefficient = sc.to_unit(self.gradient / (2 * self.msd.coords['dimensionality'].value * 192), 'cm2/s')
```

Comment:
Can you set this to the number of atoms just now rather than the hard-coded 192?